scripts/spack/configs/ubuntu20.04/packages.yaml

```diff
@@ -4,13 +4,26 @@ packages:
     - spec: mpich@3.3.2
       prefix: /usr
     buildable: false
-
+  all:
+    variants: +pic
```

Comment:
Why does ubuntu need this?

Reply:
Not sure if it does. It is just there in case the variant exists for a library and we need to make it shared. I wasn't sure if there are some system libraries that get installed on ubuntu that we use external versions for on LC machines. I can get rid of it and see if it still works.